### DISCUSSION TITLE
wifibroadcast work on openwrt

### DISCUSSION THREAD

**Discussion by zhangsd20** [svpcom/wfb-ng#235] [Questions]
I tested it on openwrt18.06 and it works very well.

But when i use wifibroadcast on openwrt19.07, its rx can't receive data.

I have also tested with openwrt18.06 and openwrt19.07 mixed.

18.06 tx     19.07 rx      its rx can't receive data
18.06 rx     19.07 tx     it work well

what could be the problem ?

**Comment by shodanx2** [comment]
Please explain, how does this work ? I have several OpenWRT routers around the house, all connected together via ethernet, I hope to fly a drone all around my house and have video reception from everywhere. Could you explain the steps to accomplish this using wfb-ng and openwrt ? 
Thanks !
